libs/platform/src/lib/components/menu-button/menu-button.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, ChangeDetectionStrategy, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { InputSize } from './../form/form-control';
+import { ButtonType } from '../button/button.component';
+import { ButtonOptions } from '../button/button.component';
+
+@Component({
+    selector: 'fdp-menu-button',
+    templateUrl: './menu-button.component.html',
+    styleUrls: ['./menu-button.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MenuButtonComponent implements OnInit {
+    /** Id for menu-button. */
+    @Input()
+    Id: string;
+
+    /** Option to make to menu-button compact. */
+    @Input()
+    size: InputSize = 'cozy';
+
+    /** Option to truncate content of the menu-button based on width. */
+    @Input()
+    width: string;
```

Comment:
Here I agree `width` and `size` is confusing how do you know as user which one is which?

Reply:
what variable should i put instead of size. please suggest?

Reply:
I have put displaySize instead of size now.